template.distillery/demo_calendar.eliom

```diff
@@ -21,6 +21,18 @@ let%server service =
 let%client service = ~%service
 
 (* A reactive value containing the currently selected date *)
+(* NOTE: in this example, we define a shared signal on the server side. It can
+   be accessed when the server generates the first page (so, declaring it
```

Comment:
> It can be accessed when the server generates the first page

Yes and no. It's value can be accessed, but it cannot be used as a true signal (i.e., updated). What about something like:

> The original value of the signal is accessible on the server while generating the first page